template-parts/degree/description/sidebar-curated.php

```diff
@@ -46,6 +47,39 @@
 		<?php endif; ?>
 
 	</div>
+<?php
+	elseif (have_rows( 'highlights_imported', $post )) : ?>
+		<div class="col-lg-6 pl-lg-5 py-lg-3 my-5">
+
+			<?php if ( have_rows( 'highlights_imported', $post ) ) : ?>
+			<h3 class="heading-underline mb-4 pb-2">Highlights</h3>
+				<?php
+				while ( have_rows( 'highlights_imported', $post ) ) : the_row();
+					$highlight_imported_icon_name = get_sub_field( 'highlight_imported_icon_name' );
+					$highlight_imported_copy      = get_sub_field( 'highlight_imported_copy' );
+
+					if ( $highlight_imported_copy ) :
+				?>
+					<div class="row mb-4">
+
+						<?php if ( $highlight_imported_icon_name ) : ?>
+						<div class="col-3 text-center">
+							<span style="font-size:5em" class="d-block text-complementary fa-solid fa-<?php echo $highlight_imported_icon_name; ?>"></span>
```

Comment:
```suggestion
							<span class="d-block text-complementary fa-5x fa-solid fa-<?php echo $highlight_imported_icon_name; ?>"></span>
```

We should always try to avoid inline styles. Font-awesome has a style for sizing.

Reply:
inline style was removed.